frontend/src/components/pages/GenerationPage/GenerationPage.js

```diff
@@ -28,22 +35,39 @@ function GenerationPage() {
       });
   }, [batchId]);
 
+  const context = {
+    batch,
+    setBatch,
+    petition,
+    setPetition,
+    ssn,
+    setSSN,
+    license,
+    setLicense,
+    attorney,
+    setAttorney,
```

Comment:
```suggestion

```

Reply:
Do we anticipate needing `setAttorney`, `setLicense`, `setBatch`, `setPetition` elsewhere in the tree? Maybe wanting to clear the inputs when we generate the petition? But I would think we'd want to keep these inputs populated--  they're not meant to change between petitions, right?

Reply:
Whoops, re-added these. They're needed by GenerationInputs to set the corresponding values.